docs/features/software-catalog/descriptor-format.md

```diff
@@ -571,3 +573,226 @@ group of people in an organizational structure.
+    picture: https://example.com/staff/jenny-with-party-hat.jpeg
+  memberOf: [team-b, backstage, ops, global-synergies, acme-corp, employees]
+  directMemberOf: [team-b, employees]
+```
+
+In addition to the [common envelope metadata](#common-to-all-kinds-the-metadata)
+shape, this kind has the following structure.
+
+### `apiVersion` and `kind` [required]
+
+Exactly equal to `backstage.io/v1alpha1` and `User`, respectively.
+
+### `spec.type` [required]
+
+The type of user as a string, e.g. `employee`. There is currently no enforced
+set of values for this field, so it is left up to the adopting organization to
+choose a nomenclature that matches their staff roles. It is recommended that
+this field be used in an organizational sense, rather than for separating out
+different roles that employees may have.
+
+Some common values for this field could therefore be:
+
+- `employee`
+- `contractor`
+
+Some examples of values that are a poor fit for this field, that may be modeled
+in other fields:
+
+- `developer`
+- `cook`
+- `terminated`
+
+### `spec.status` [required]
+
+The status of the user as a string, e.g. `active`. There is currently no
+enforced set of values for this field, so it is left up to the adopting
+organization to choose a nomenclature that matches their staff statuses. It is
+recommended that this field be used in an organizational/employment sense,
+rather than for separating out different roles that employees may have or
+transient short term statuses.
+
+Some common values for this field could therefore be:
+
+- `active`
+- `pending`
+- `terminated`
+
+Some examples of values that are a poor fit for this field, that may be modeled
+in other fields:
+
+- `on-vacation`
+- `developer`
+
+### `spec.profile` [optional]
+
+Optional profile information about the user, mainly for display purposes. All
+fields of this structure are also optional. The email would be a primary email
+of some form, that the user may wish to be used for contacting them. The picture
+is expected to be a URL pointing to an image that's representative of the user,
+and that a browser could fetch and render on a profile page or similar.
+
+### `spec.memberOf` [required]
+
+The list of groups that the user is a direct or indirect member of (i.e., the
+groups that the user is a direct member of, plus all of the recursive parent
+groups of those groups). The list must be present, but may be empty if the user
+is not member of any groups. The items are not guaranteed to be ordered in any
+particular way.
+
+The entries of this array are
+[entity references](https://backstage.io/docs/features/software-catalog/references),
+with the default kind `Group` and the default namespace equal to the same
+namespace as the user. Only `Group` entities may be referenced. Most commonly,
+these entries point to groups in the same namespace, so in those cases it is
+sufficient to enter only the `metadata.name` field of those groups.
+
+### `spec.directMemberOf` [required]
```

Comment:
Thinking we should only have `memberOf` and let the rest be discoverable through relationships? And ofc have that be only direct membership

Reply:
Sure - the other one was only added as an analogue to what we already had for `Group`. Shall I remove the transitive one and just assume that the relationships concept solidifies in some shortish term?

Reply:
Yeah I think so, got an opportunity to clean up the Group kind too here